components/resources/brave_shields_strings.grdp

```diff
@@ -13,24 +13,24 @@
   </message>
 
   <message name="IDS_BRAVE_SHIELDS_UP" desc="Status label for when Brave Shields is enabled.">
-    Shields are UP <ph name="LINK_BEFORE">$1</ph>for<ph name="LINK_AFTER">$2</ph>
+    <ph name="STATUS_BEFORE">$1</ph>Shields are UP<ph name="STATUS_AFTER">$1</ph> for <ph name="HOST">$2</ph>
   </message>
 
   <message name="IDS_BRAVE_SHIELDS_DOWN" desc="Status label for when Brave Shields is disabled.">
-    Shields are DOWN <ph name="LINK_BEFORE">$1</ph>for<ph name="LINK_AFTER">$2</ph>
+    <ph name="STATUS_BEFORE">$1</ph>Shields are DOWN<ph name="STATUS_AFTER">$1</ph> for <ph name="HOST">$2</ph>
   </message>
 
   <message name="IDS_BRAVE_SHIELDS_BROKEN" desc="A footnote below the main Shields switch button.">
     If this site seems broken, try Shields down.
-    <ph name="LINK_BEFORE">$1</ph>Note: this may reduce Brave's privacy protections.<ph name="LINK_AFTER">$2</ph>
+    <ph name="NOTE_BEFORE">$1</ph>Note: this may reduce Brave's privacy protections.<ph name="NOTE_AFTER">$1</ph>
```

Comment:
Do we even need placeholders here since the entire string is wrapper in them? Wouldn't it be easier to just put `<span>` in to the `render`?

Reply:
Its two strings, with a newline :smile: - now if you  were asking whether these should be two separate strings.......

I did think about that but figured I was already changing quite a lot.

Reply:
Oh lol, I didn't even notice the first line 🤦‍♂️ never mind 😄